src/runtime/current_thread/runtime.rs

```diff
@@ -8,6 +8,9 @@ use futures::Future;
 
 use std::io;
 
+// Re-export Handle from here
+pub use executor::current_thread::Handle;
```

Comment:
For forwards compat, I would wrap this in a new type.

Reply:
Done